docker/k8s/sgx-enclave-manager-deployment.yaml

```diff
@@ -16,21 +16,21 @@ version: '3.5'
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: enclave-manager-pool
+  name: avalon-enclave-manager-sgx
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: tee
+      app: enclave-manager-sgx
   template:
     metadata:
-      name: tee
+      name: enclave-manager-sgx
```

Comment:
Can we use the naming convention same as in docker, change to  enclave-manager-sgx to enclave-manager. So that there is consistency throughout.

Reply:
This is in order to distinguish enclave manger container with and without SGX support.  I just noticed that the naming convention for enclave manger docker file with SGX support need to change. The docker image  for enclave manager with and without SGX support should be different. @manju956 could you please take care of it